src/openms/include/OpenMS/FORMAT/HANDLERS/MzMLHandler.h

```diff
@@ -2011,8 +2011,33 @@ namespace OpenMS
             chromatogram_.getPrecursor().getPossibleChargeStates().push_back(value.toInt());
           }
         }
+        else if (accession == "MS:1002476") //ion mobility drift time
+        {
+          // Drift time may be a property of the precursor (in case we are
+          // acquiring a fragment ion spectrum) which is consistent with the CV
+          // annotation which lists MS:1002476 with the relation "is_a:
+          // MS:1000455" which is an ion selection attribute. Consequently,
+          // this property should thus go into the selectedIon tag inside a
+          // <precursor>
+          //
+          // However, drift time may also be a property of the scan itself
+          // (similar to retention time).
+
+          // This assumes that this is a fragment ion spectrum derived from a precursor with a given drift time
+          if (in_spectrum_list_)
+          {
+            spec_.getPrecursors().back().setDriftTime(value.toDouble());
+            spec_.setDriftTime(value.toDouble());
```

Comment:
one question here is whether we should *only* store it in the precursor, thus allowing us to reproduce 1:1 the input file (and not add new CV terms). However, this may make it harder for software that tries to use the `getDriftTime` interface of MSSpectrum ... another option is to change  `getDriftTime` MSSpectrum to check whether any underlying precursors have a drift time